syntax = "proto3";

package fizzbuzz;

message FizzBuzzDFRequest {
	string col_name = 1;
}

message FizzBuzzDFReply {
	string df = 1;
}

// computes the fizzbuzz
service Computator {
	rpc ComputeFizzBuzz (FizzBuzzDFRequest) returns (FizzBuzzDFReply) {}
}